### PR TITLE
fix(net6): Adjust keyboard location computation

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/8703

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?
Restores proper occluded keyboard rectangle computation.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/Controls/BindableNSView.macOS.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ public BindableNSView()
 			}
 		}
 
-		public BindableNSView(IntPtr handle)
+		public BindableNSView(ObjCRuntime.NativeHandle handle)
```
Comment (spouliot) [inline src/Uno.UI/Controls/BindableNSView.macOS.cs:62]: Does this need to remain compatible with the legacy SDK ? If so

```csharp
using ObjCRuntime;
#if !NET
using NativeHandle = System.IntPtr;
#endif
```
and use `NativeHandle` elsewhere in the file

File: src/Uno.UI/UI/Xaml/UIElement.wasm.cs [hunk 2]
```diff
@@ -317,7 +317,7 @@ protected internal void SetHtmlContent(string html)
 			);
 		}
 
-		internal static UIElement GetElementFromHandle(IntPtr handle)
+		internal static UIElement GetElementFromHandle(ObjCRuntime.NativeHandle handle)
```
Comment (spouliot) [inline src/Uno.UI/UI/Xaml/UIElement.wasm.cs:320]: inside a `*.wasm.cs` file ?

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/UIElement.wasm.cs]: Yep, that was a find/replace gone bad ;)

File: src/Uno.UI/Controls/BindableNSView.macOS.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI/Controls/BindableNSView.macOS.cs]: Indeed, fixed in a later change. CI was not happy about it ;)

**Review by MartinZikmund — APPROVED** [review]

**Pull request merged**